solutions/cc-AMONGUS2.mdx

```diff
@@ -0,0 +1,109 @@
+---
+id: cc-AMONGUS2
+source: CC
+title: Among Us
+author: Ryan Chou
+---
+
+## Explanation
+If we only have two astronauts $x$ and $y$, then we have 4 cases possible cases. 
+
+* If $x$ vouches for $y$ and $x$ is a parasite, then $y$ must also be a parasite.
+
+* If $x$ vouches for $y$ and $x$ is a human, then $y$ must also be a human.
+
+* If $x$ accuses $y$ of being a parasite and $x$ is a parasite, then $y$ must be a human.
+
+* If $x$ accuses $y$ of being a parasite and $x$ is a human, then $y$ must be a parasite.
```

Comment:
(optional) replace astronaut with crewmate and parasite with impostor

Reply:
um, ok?

Reply:
lol this was a joke